docs/rfc/0003_vex_support.md

```diff
@@ -0,0 +1,218 @@
+
+### User story #1
+
+As a user, I want to scan registries on my infrastracture filtering out the 
+number of CVEs detected by SBOMbastic, removing false positives as much as possible.
+
+### User story #2
+
+As a user, I want to configure the registries scan with appropriate VEX files 
+depending on the content of the registry (eg. test, staging or prod images), 
+so that the vulnerability report will have an accurate result.
+
+# Detailed design
+
+[design]: #detailed-design
+
+<!---
+This is the bulk of the RFC. Explain the design in enough detail for somebody
+familiar with the product to understand, and for somebody familiar with the
+internals to implement.
+
+This section should cover architecture aspects and the rationale behind
+disruptive technical decisions (when applicable), as well as corner-cases and
+warnings.
+--->
+
+This RFC introduces support for VEX files when scanning registries.
+From now on, the Registry CRD will have an additional field to refer the VexHub
+resources we want to use to scan it.
+
+Suppose you are scanning the registry of a dedicated department in your company.
+This registry will host specific kind of images depending on the scope of the 
+department. In order to make the scan as much accurate as possible, we are 
+going to provide a new clusterwide CRD called VexHub. This CRD will hold 
+configuration of the VexHub repositories, so that the Registry CRD will refer 
+to these VexHub resources.
+
+Since VexHub CRD is clusterwide, this means that you can use the same 
+configuration across multiple registries.
+
+## VexHub CRD
+
+To configure a new VexHub, the user will need to apply a new manifest as follow:
+
+```yaml
+apiVersion: sbombastic.rancher.io/v1alpha1
+kind: VexHub
+metadata:
+  name: vendor-vexhub
+  namespace: default
+spec:
+  url: https://vexhub.vendor.com
+  credentials:
+    secretRef:
+      name: vendor-vexhub-secret
+      key: access-token
+```
+
+As you can see the VexHub CRD has a `secretRef` field, which means that you 
+need to provide a Secret to configure the credentials (if any):
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vexhub-auth
+  namespace: security-team
+type: Opaque
+data:
+  token: <base64-encoded-token>
+
+```
+
+## Registry CRD
+
+To configure the Registry to use the VexHub repository, the user will need to 
+update the Registry manifest as follow:
+
+```yaml
+apiVersion: scanner.rancher.io/v1alpha1
+kind: Registry
+metadata:
+  name: registry-example
+  namespace: default
+spec:
+  url: "https://registry-1.docker.io"
+  type: "docker"
+  auth:
+    secretName: "registry-secret"
+  discoveryPeriod: "1h"
+  scanPeriod: "1d"
+  repositories:
+    - "repo1"
+    - "repo2"
+  vexHubRefs:
+    - name: vendor-vexhub
+    - name: internal-vexhub # example of multiple VexHub configuration
+```
+
+This way the registry will be scanned with appropriate VEX files.
+
+## AirGap
+
+AirGap is available by default for this feature, since the only requirement is
```

Comment:
add link to the airgap vexhub guide https://github.com/aquasecurity/trivy/blob/main/docs/docs/advanced/air-gap.md#vex-hub